rust/core-lib/src/find.rs

```diff
@@ -0,0 +1,304 @@
+use std::cmp::{min,max};
```

Comment:
Also, underneath the copyright notice, all of our modules (hopefully) have a very short (often one line) //! top level doc comment; here that would be something like "Logic and state management for search and replace"

Reply:
I can add it. It seems there are several modules and public types that don't have the doc or top level doc comment. Like `View` and `Editor`.

Reply:
Yea, this may be missing from some older files, but we've been trying to get more consistent with it with new files.